docs/application/web/api/5.5/ui_fw_api/Wearable_UIComponents/wearable_circularindexscrollbar.htm

```diff
@@ -14,6 +14,13 @@ <h1>Circular Index Scroll Bar</h1>
 <p>If you spin the rotary quickly, an indicator will be displayed and you can move in an index unit.</p>
 <p>The indicator will disappear if no rotary action is detected for a second.</p>
 <p>This component fires a select event when the index characters are selected.</p>
+<button
```

Comment:
Just for check.
Unlike other pages, in this page, the image location is under the note.
Is it intended?
![image](https://user-images.githubusercontent.com/9958374/71387122-98637b00-2635-11ea-900a-9ed914602d34.png)

Reply:
improved order